demoextendsymfonyform1/demoextendsymfonyform1.php

```diff
@@ -24,34 +21,32 @@
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
 use PrestaShopBundle\Form\Admin\Type\YesAndNoChoiceType;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
 
 /**
- * Class DemoExtendSymfonyForm1 demonstrates the usage of CQRS pattern and hooks.
+ * Class DemoExtendSymfonyForm1 demonstrates the usage of Symfony hooks.
  */
 class DemoExtendSymfonyForm1 extends Module
 {
     public function __construct()
     {
         $this->name = 'demoextendsymfonyform1';
-        $this->version = '1.0.0';
-        $this->author = 'Tomas Ilginis';
+        $this->version = '1.1.0';
+        $this->author = 'PrestaShop';
         $this->need_instance = 0;
 
         parent::__construct();
 
         $this->displayName = $this->getTranslator()->trans(
             'Demo Symfony Forms #1',
             [],
-            'Modules.Democqrshooksusage.Admin'
+            'Modules.DemoHowToExtendSymfonyForm.Admin'
```

Comment:
```suggestion
            'Modules. DemoExtendSymfonyForm1.Admin'
```

shouldn' this be like that? have you checked translations?

Reply:
Ah, you mean if I don't pick the right name it will not be translatable inside the BO interface?

To be honest I did not wanted to make it translatable, it was just an example of how to do the php part 😅

Reply:
I'll keep this in mind 😉  but for another PR as translation is... a so much scarier topic 😅